README.md

```diff
@@ -134,6 +134,7 @@ config
 
```

Comment:
An entry in CHANGES_NEXT_RELEASE describing the change introduced by this PR should be added.

Reply:
Fixed in 13472fb50f0a1bfd3becd57d21736c5ee4dd2248